lms/djangoapps/instructor_task/api.py

```diff
@@ -280,6 +280,11 @@ def submit_bulk_course_email(request, course_key, email_id):
     # the InstructorTask status.
     email_obj = CourseEmail.objects.get(id=email_id)
     targets = [target.target_type for target in email_obj.targets.all()]
+    #task_input has a limit to the size it can store, so cohorts are combined and counted
+    num_cohorts = targets.count('cohort')
+    if num_cohorts > 1:
+        targets.remove('cohort')
```

Comment:
you might want to do a list comprehension here to filter out the `"cohort"`s in this list, since `remove` will only remove the first instance of `"cohort"`:
```python
In [5]: lst = ['cohort', 1, 'cohort', 2, 'cohort', 3]
In [6]: lst.remove('cohort')
In [7]: lst
Out[7]: [1, 'cohort', 2, 'cohort', 3]
```